datadog/dogshell/common.py

```diff
@@ -10,23 +10,31 @@
 
 def print_err(msg):
     if is_p3k():
-        print('ERROR: ' + msg + '\n', file=sys.stderr)
+        print(msg + '\n', file=sys.stderr)
     else:
         sys.stderr.write(msg + '\n')
 
 
 def report_errors(res):
     if 'errors' in res:
-        for e in res['errors']:
-            print_err('ERROR: ' + e)
+        errors = res['errors']
+        if isinstance(errors, list):
+            for error in errors:
+                print_err("ERROR: " + error)
```

Comment:
My bad, let's use `format` everywhere as well, so that there are no problems with types when trying to do the `+` operation